SwiftUI_DZ/SwiftUI_DZ/View/StartedView.swift

```diff
@@ -0,0 +1,174 @@
+// StartedView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран старта
+struct StartedView: View {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let titleText = "online furniture store"
+        static let getStartedButtonText = "get started"
+        static let haveAccountText = "Don't have an account?"
+        static let singButtonText = "Sing in here"
+        static let vStackSpacing: CGFloat = 20
+        static let descriptionViewTagString = "descriptionView"
+        static let singUpViewTagString = "singUpView"
+        static let wifiSystemImageName = "wifi"
+        static let colorOpacity: CGFloat = 0.9
+        static let emptyString = ""
+        static let phaseImageWidth: CGFloat = 200
+        static let phaseImageHeight: CGFloat = 150
+        static let buttonRectangleCornerRadius: CGFloat = 30
+        static let purpleOpacity: CGFloat = 0.6
+        static let getStartedButtonRectangleFrame: CGFloat = 250
+        static let singHereRectangleWidth: CGFloat = 150
+        static let singHereRectangleHeight: CGFloat = 2
+        static let singHereButtonPadding: CGFloat = 50
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        NavigationView {
+            VStack(spacing: Constants.vStackSpacing) {
+                Spacer()
+                titleTextView
+                furnitureImageView
+                Spacer()
+                getStartedButtonView
+                navigationToDescriptionView
+                haveAccountTextView
+                singHereButtonView
+                NavigationLink(
+                    destination: AuthentificationView(),
+                    tag: Constants.singUpViewTagString,
+                    selection: $tagSelection
+                ) { EmptyView() }
+            }
+            .frame(
+                width: UIScreen.main.bounds.width,
+                height: UIScreen.main.bounds.height
+            )
+            .background(gradientColor)
+            .ignoresSafeArea(.container, edges: .all)
+        }
+    }
+
+    // MARK: - Private Properties
+
+    @State private var tagSelection: String?
+
+    private let furnitures = Furniture.getFurniture()
```

Comment:
```suggestion
    @State private var tagSelection: String?

    private let furnitures = Furniture.getFurniture()
```
     private let 
    @Environment private var 
    @ObservedObject private var 
    @StateObject private var 
    @Binding private var 
    @State private var 
    private var